openwisp_radius/social/views.py

```diff
@@ -55,7 +55,7 @@ def get_redirect_url(self, request):
         Token.objects.filter(user=user).delete()
         RadiusToken.objects.filter(user=user).delete()
         token = Token.objects.create(user=user)
-        rad_token = RadiusToken.objects.create(user=user)
+        rad_token = RadiusToken.objects.create(user=user, organization=organization)
```

Comment:
what if a token already exists?

Reply:
Token is deleted 2 lines above the highlighted line, however in interest of reusing code, I am going for `get_or_create_radius_token` here as well! :smile: